source/FreeRTOS_IP.c

```diff
@@ -190,6 +178,10 @@ uint16_t usPacketIdentifier = 0U;
  * reference. */
 const MACAddress_t xBroadcastMACAddress = { { 0xff, 0xff, 0xff, 0xff, 0xff, 0xff } };
 
+/* __XX__ temporary for coverity build */
+/** @brief Structure that stores the netmask, gateway address and DNS server addresses. */
+NetworkAddressingParameters_t xNetworkAddressing = { 0, 0, 0, 0, 0 };
+
```

Comment:
Why is this needed ?

Reply:
Do you mean why it is initialised, or what the function is of `xNetworkAddressing`?
It is initialised just to show it explicitly.
The struct will be dropped as soon as we have multiple interfaces. The struct will be replaced by end-points.

Reply:
Yes hein, as we have moved to end point based implementation , i removed it in previous PR, was just checking why was it needed again for fixing coverity.